http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClient.java

```diff
@@ -380,6 +383,17 @@ NettyConfiguration configuration() {
          * @return The builder for method chaining.
          */
         Builder tlsKeyManagersProvider(TlsKeyManagersProvider keyManagersProvider);
+
+        /**
+         * Sets initial window size of a stream. This setting is only respected when the HTTP/2 protocol is used.
+         *
+         * See <a href="https://tools.ietf.org/html/rfc7540#section-6.5.2">https://tools.ietf.org/html/rfc7540#section-6.5.2</a>
+         * for more information about this parameter.
+         *
+         * @param initialWindowSize The initial window size of a stream.
+         * @return This builder for method chaining.
+         */
+        Builder initialWindowSize(Integer initialWindowSize);
```

Comment:
It'd be nice to group all h2 settings in one configuration class and it might be too late to do that.

Do you think if it makes sense to add `h2` in the name something like `initialHttp2WindowSize` to make it more clear this only applies to h2? My concern is we have growing number of configurations and it might be confusing to customers.

Reply:
Yeah, I think I would have preferred that as well, but we already have `maxStreams` as a top level setter. Since it's the only one for now though, it might be fine to deprecate it and introduce an `Http2Config` now. What do you think?

Reply:
Sounds good to me!